### PR TITLE
its != it's 😊

### PR DESCRIPTION
**Pull request by Chupaka**
## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_posts/2018-10-12-release-80.markdown [hunk 1]
```diff
@@ -169,7 +169,7 @@ Experiencing issues introduced by this release? Please report them in our [issue
 ## All changes
 
 - Add mode (daily/hourly) to darksky ([@randellhodges] - [#16719]) ([weather docs]) ([weather.darksky docs]) ([weather.openweathermap docs])
-- Extract lovelace to it's own component ([@balloob] - [#16816]) ([frontend docs]) ([lovelace docs])
```
Comment (raman325) [inline source/_posts/2018-10-12-release-80.markdown]: While grammatically correct, this is the title of a PR and it should therefore not be changed I think

File: source/_posts/2018-11-28-release-83.markdown [hunk 2]
```diff
@@ -244,7 +244,7 @@ Experiencing issues introduced by this release? Please report them in our [issue
 - Add Xiaomi Air Purifier 2s support ([@syssi] - [#18260]) ([fan.xiaomi_miio docs])
 - Allow alexa to simply turn on and off climate components. ([@Parlane] - [#16989]) ([alexa docs])
 - Enable config flow for Luftdaten ([@fabaff] - [#17700]) ([luftdaten docs]) ([sensor.luftdaten docs]) (breaking change)
-- Adding current_humidity to attributes if its not None. ([@kennedyshead] - [#18261]) ([climate docs])
+- Adding current_humidity to attributes if it's not None. ([@kennedyshead] - [#18261]) ([climate docs])
```
Comment (raman325) [inline source/_posts/2018-11-28-release-83.markdown]: PR title

File: source/_posts/2019-01-23-release-86.markdown [hunk 3]
```diff
@@ -129,7 +129,7 @@ Experiencing issues introduced by this release? Please report them in our [issue
 ## Breaking Changes
 
 - Automation: Split time_pattern triggers from time trigger ([@armills] - [#19825]) ([automation.interval docs]) ([automation.time docs]) (breaking change)
-- Each instance of Home Assistant will now generate it's own unique webhook url for Geofencyto use. One will need to be generated and provided to the iOS app to replace the existing url: `https://myhome.duckdns.org/api/geofency?api_password=HA_PASSWORD` ([@rohankapoorcom] - [#18951]) ([geofency docs]) (breaking change)
+- Each instance of Home Assistant will now generate its own unique webhook url for Geofencyto use. One will need to be generated and provided to the iOS app to replace the existing url: `https://myhome.duckdns.org/api/geofency?api_password=HA_PASSWORD` ([@rohankapoorcom] - [#18951]) ([geofency docs]) (breaking change)
```
Comment (raman325) [inline source/_posts/2019-01-23-release-86.markdown]: PR title

File: source/_posts/2019-01-23-release-86.markdown [hunk 4]
```diff
@@ -140,8 +140,8 @@ Experiencing issues introduced by this release? Please report them in our [issue
...
+- The gpslogger device_tracker platform no longer takes any configuration. The configuration needs to be applied to the component gpslogger instead. The platform will be automatically loaded with the component and should not be specified in configuration.yaml. Each instance of Home Assistant will now generate its own unique webhook for GPSLogger to use. One will need to be generated and provided to the mobile app to the replace the existing url: `https://myhome.duckdns.org/api/gpslogger?api_password=HA_PASSWORD`. Home Assistant only supports using POST (no longer GET) for GPSLogger. The long lived access token is also no longer needed. ([@rohankapoorcom] - [#20044] - [#20079]) ([device_tracker docs]) ([gpslogger docs]) (breaking change)
```
Comment (raman325) [inline source/_posts/2019-01-23-release-86.markdown]: PR title

**Review by raman325 — COMMENTED** [review]
I'm not going to go through all of them but there are enough here that hopefully you get the point. If you'd like, we can wait for another contributor to weigh in as I am not 100% confident in my original comment.

With that being said, all of the changes in `_docs/configuration`, `_faq`, and `_integrations` made sense to me, nice find!

**Comment by janiversen** [comment]
I think we need to decide which flavor of english do we use (is it color or colour), changing “its” to “it’s” is pending on the decision.

According to the oxford dictionary:
https://www.oxfordlearnersdictionaries.com/definition/english/its?q=its
used to educate children, “its” is perfectly legal.

Just for information “it’s” is in the same dictionary:
https://www.oxfordlearnersdictionaries.com/definition/english/it-s?q=it%E2%80%99s

**Comment by MartinHjelmare** [comment]
It's not a matter of English flavor. `it's` and `its` have different grammatical meaning. The former is a short form for `it is` or `it has`, while the latter is a possessive adjective.

File: source/_posts/2018-11-28-release-83.markdown (hunk 2 above)
Comment (frenck) [reply source/_posts/2018-11-28-release-83.markdown]: Agreed, please revert this, this is generated content.

**Comment by janiversen** [comment]
@MartinHjelmare you are right, I mixed issues, but replacing “its” with “it´s” in a generated manner is still wrong, you need to control every single change.

**Comment by Chupaka** [comment]
Okay, reverted all changes where PR numbers are mentioned

**Comment by Chupaka** [comment]
@janiversen that was not generated, I carefully read all occurrences - it was fully manual. There are changes in both directions: its -> it's and it's -> its, they are in different commits

**Comment by janiversen** [comment]
Thanks for the explanation, it was not obvious. In this case I am happy 😃

**Comment by raman325** [comment]
just to add, for the folders I mentioned, I reviewed every change contextually to ensure it made sense

**Review by raman325 — APPROVED** [review]
Looked through `_posts` too and every change looks contextually accurate. Thanks again!

**Review by frenck — APPROVED** [review]
Thanks, @Chupaka 👍

**Pull request merged**